cypress/support/pages/home.pageObject.js

```diff
@@ -4,7 +4,47 @@ class HomePageObject extends PageObject {
   url = '/#/';
 
   get usernameLink() {
-    return cy.getByDataCy('username-link');
+    return cy.getByDataQa('username-link');
+  }
+
+  assertUsername(username) {
+    this.usernameLink.should('contain', username);
+  }
+
+  get globalFeed() {
+    return cy.getByDataQa('Global-Feed');
+  }
+
+  get NoArticleMessage() {
```

Comment:
The method name 'NoArticleMessage' should start with a lowercase letter. Consider renaming it to 'noArticleMessage'.